presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduRecordCursor.java

```diff
@@ -14,19 +14,26 @@
 package io.prestosql.plugin.kudu;
```

Comment:
Can you please add a tests where incorrect column mapping was exposed?

Reply:
AbstractTestQuery#testCorrelatedJoin, testTableSampleWithFiltering, testSelectAllFromOuterScopeTable exposes this incorrect table mapping. `TestKuduDistributedQueries`  captures most of the cases where incorrect column mapping was exposed